lib/rules/jsdoc.js

```diff
@@ -22,8 +22,8 @@
  *
  * There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
  *
- * By default it allows any tag of mixed set. You can pass `Object`
- * to select preset with `preset` field and add custom tags with `extra` field.
+ * By default it allows any tag of mixed set. You can pass `Object` to select
+ * preset with `preset` field and add custom tags with `extra` field.
```

Comment:
I guess you are just saying it allows all the tags from the preset you picked as well as any you add in the extra field?